test_scripts/Defects/8_0/2399__SDL_respond_UNSUPPORTED_RESOURCE_in_case_SDL_4_0_was_omited.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2399
+---------------------------------------------------------------------------------------------------
+-- Description:
+-- SDL does respond UNSUPPORTED_RESOURCE to SystemRequest(QUERY_APPS) in case app is registered with protocol version
+-- less than 4.0
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- 1) App is registered and activated
+-- 2) Mobile app requests SystemRequest(QUERY_APPS)
```

Comment:
@VNikiforov-dev Please use one of `App` or `Mobile app` in bound of whole script